src/main/java/com/endava/soa4/PropertyLoader.java

```diff
@@ -0,0 +1,14 @@
+package com.endava.soa4;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class PropertyLoader {
+    public static String getProperties(String key) throws IOException {
+        Properties prop = new Properties();
+        InputStream ip = ClassLoader.getSystemResourceAsStream("config/config.properties");
```

Comment:
Please close inputStream after you finished to use it.

Reply:
changed it to try-with-resources